### ISSUE TITLE
Clarify the version of the QSTEM source

### ISSUE THREAD

**Issue by ericpre** [conda-forge/pyqstem-feedstock#1]
As mentioned in https://github.com/conda-forge/staged-recipes/pull/8656#issuecomment-508017655, there are two things not right in this package regarding the QSTEM source:
- What is the version of the qstem source code (which is vendored in this package),
- It would be better to pull the QSTEM source from the official repository, which should be https://github.com/QSTEM/QSTEM

**Comment by schiotz** [comment]
@jacobjma I agree with @ericpre here: we should make it clear which version of QSTEM went into pyqstem, just as a matter of good scientific practice.  Have you records of which version of QSTEM you used when you made pyqstem?  Then we can put it into some variable in the main module, I guess.

As for the second point, I respectfully disagree: The pyqstem github includes everything it needs, and installing something else than what is in the github would in my opinion be breaking peoples' expectations.

Furthermore, the QSTEM github seems not to be up to date, it has not been updated in five years, and there have been later releases of the QSTEM program - but perhaps only the gui has changed.  Do you know anything about this, @jacobjma?  Did you discuss which versions of the source code (and how to keep it up to date) with Christian Koch while you were staying at his group?

**Comment by jacobjma** [comment]
PyQSTEM is based on the latest QSTEM source code on github (the one updated about five years ago). Interfacing this source with python required some changes that could break the GUI version, at least it is not tested. Pulling the source code from this repository would require some work.

The original developer Christoph Koch maintains a slightly different version of the source. He talked about merging his version with the one on github, but I think the versions diverged quite a few changes ago.

**Comment by schiotz** [comment]
Maybe it would make sense to add two variables to the pyqstem main module:

pyqstem.version = '2.0.3'
pyqstem.qstem_version = 'efc247315f79c09eab382b1c1db80d507019e7ff'

The officially released version 2.30 seems to be around a year older than the latest commit to github.

@ericpre Does this suggestion makes sense?

**Comment by ericpre** [comment]
Yes, documenting it would be a good first step, but the best would be to have one source code for QSTEM! I understand this is not all in our hand but we need to encourage the best practise.

> As for the second point, I respectfully disagree: The pyqstem github includes everything it needs, and installing something else than what is in the github would in my opinion be breaking peoples' expectations.

Even if this would need a bit of work to set it up properly, I think this is a "technical detail" and there are ways to get code from one repository into another, by using git submodule, for example. There may be other approach to consider or more appropriate to deal with this.

> Interfacing this source with python required some changes that could break the GUI version, at least it is not tested. Pulling the source code from this repository would require some work.

If I understand correctly, it means that the forked QSTEM source code included in pyqstem has been modified to work with pyqstem?

**Comment by jacobjma** [comment]
Yes, the forked QSTEM source code has been modified to work with PyQSTEM.
